charts/kube-prometheus-stack/templates/extra-objects.yaml

```diff
@@ -1,4 +1,15 @@
-{{ range .Values.extraManifests }}
+{{- /* Normalize extraObjects to a list, easier to loop over */ -}}
+{{- $extraObjects := .Values.extraManifests | default (list) -}}
+
+{{- if kindIs "map" $extraObjects -}}
+  {{- $extraObjects = values $extraObjects -}}
+{{- end -}}
+
+{{- range $extraObjects }}
 ---
-{{ tpl (toYaml .) $ }}
-{{ end }}
+  {{- if kindIs "map" . }}
+    {{- tpl (toYaml .) $ | nindent 0 }}
+  {{- else if kindIs "string" . }}
+    {{- tpl . $ | nindent 0 }}
```

Comment:
what is the use case for `nindent 0`

Reply:
it just allows me to indent the if block, nothing else
```
  {{- if kindIs "map" . }}
    {{- tpl (toYaml .) $ | nindent 0 }}
  {{- else if kindIs "string" . }}
```